flagging_site/app.py

```diff
@@ -42,14 +43,33 @@ def create_app(config: Optional[Config] = None) -> Flask:
     from . import blueprints
     register_blueprints_from_module(app, blueprints)
 
+    #Swagger configs for flasgger
+    swagger_config = {
+        "headers": [
+        ],
+        "specs": [
+            {
+                "endpoint": 'apispec_1',
+                "route": '/apispec_1.json',
+                "rule_filter": lambda rule: True,  # all in
+                "model_filter": lambda tag: True,  # all in
+            }
+        ],
+        "static_url_path": "/flasgger_static",
+        # "static_folder": "static",  # must be set by user
+        "swagger_ui": True,
+        "specs_route": "/api/docs"
```

Comment:
I'm going to come off as nitpicky here but: make sure it's single quotes, not double, for consistency with the rest of the project.

Reply:
Ah yes. My bad, I was copying and pasting it from the flasher documentation. I will fix this later tonight.

Reply:
Got rid fo the double quotes.